BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -368,6 +368,9 @@ public final class ConvivaAnalytics: NSObject {
             // for live stream, until bitmovin expose `AVERAGE-BANDWIDTH` from the m3u8 file, we will report bitrate as the average bitrate
             // videoAnalytics.reportPlaybackMetric(CIS_SSDK_PLAYBACK_METRIC_AVERAGE_BITRATE, value: bitrate)
             videoAnalytics.reportPlaybackMetric(CIS_SSDK_PLAYBACK_METRIC_BITRATE, value: bitrate)
+            if player.isAd {
+                adAnalytics.reportPlaybackMetric(CIS_SSDK_PLAYBACK_METRIC_BITRATE, value: bitrate)
```

Comment:
confirming should the method be `.reportAdMetric`?

Reply:
fixed

Reply:
also use it in `onAdPlaybackStateChanged`. The previous calls seem working but we should follow their documentation